src/Compilers/CSharp/Test/Emit/PDB/PDBLambdaTests.cs

```diff
@@ -1296,6 +1296,156 @@ void F()
 </symbols>");
         }
 
+        [Fact]
+        public void SwitchExpressionWithLambda1()
+        {
+            string source = WithWindowsLineBreaks(@"
+using System;
+
+class C
+{
+    static string M(object o)
+    {
+        return o switch
+        {
+            int i => new Func<string>(() => $""Number: {i} + {o} == {i + (int)o}"")(),
+            _ => ""Don't know""
+        };
+    }
+}
+");
+
+            // we just want this to compile without crashing/asserting
```

Comment:
>// we just want this to compile without crashing/asserting [](start = 12, length = 58)

I'd remove the comment. We do verify the generated PDB, so it's not just that it compiles.